tests/Rector/Class_/RenameClassRector/RenameClassRectorTest.php

```diff
@@ -17,17 +18,29 @@
  */
 final class RenameClassRectorTest extends AbstractRectorTestCase
 {
-    public function test(): void
+    /**
+     * @dataProvider provideTestFiles
+     */
+    public function test(string $filePath): void
+    {
+        $this->doTestFiles([$filePath]);
```

Comment:
This look kinda hacky. 

First extract `$this->doTestFile()` in `AbstractRectorTestCase` in standalone PR before we merge this

Reply:
Too much work :-) i hope you will be okay if i do it directly in this PR